lib/websocket_server.rb

```diff
@@ -2,6 +2,8 @@ class WebsocketServer
   Pairing = Struct.new(:is_ws, :proxy)
 
   def initialize
+    @logger = Rails.env.production? ? $websocket_log : Logger.new(STDOUT)
```

Comment:
We (UI developers) ususally don't start the whole EVM for development, that's why I'm mounting the websocket server as a route in development: https://github.com/ManageIQ/manageiq/blob/master/config/routes.rb#L2654